src/lib/InvoiceForm.svelte

```diff
@@ -18,23 +19,25 @@ const validImageTypes = ["image/jpg", "image/jpeg", "image/png"];
 const mb = 1 << (10 * 2);
 const dayInMs = 1000 * 60 * 60 * 24;
 
-let notifications: Array<string> = [];
+let notifications: Array<string> = $state([]);
 
-$: if (browser) {
-    // total amount tracker
-    $invoice.totalAmount = 0;
-    for (const row of $invoice.rows) {
-        if (row.rate === undefined || row.quantity === undefined) {
-            row.amount = "";
-            continue;
-        }
+run(() => {
```

Comment:
they broke their API in v5

Reply:
maybe relying on reactivity (essentially what sold svelte compiler to so many people) was not a good idea for this task, we live and we learn.

Same code can not be replicated by new effect(() => {}) hook which is the new $: () =? {} but react version of everything, in svelte 4, everything was working without warnings and errors and store tracking also worked here, in svelte 5 new effect(() => {}) hook doesn't work and doesn't track the store